examples/arguments.rs

```diff
@@ -0,0 +1,42 @@
+extern crate radeco_lib;
+extern crate r2pipe;
+extern crate r2api;
+
+use r2api::api_trait::R2Api;
+use r2pipe::R2;
+
+use radeco_lib::middle::regfile::SubRegisterFile;
+use radeco_lib::frontend::radeco_containers::{ProjectLoader, ModuleLoader, FunctionLoader};
+use radeco_lib::frontend::radeco_source::Source;
+use radeco_lib::analysis::functions::{FunctionAnalyzer, FuncArgumentAnalyzer};
+
+use std::cell::RefCell;
+use std::rc::Rc;
+
+fn main() {
+    let mut r2 = R2::new(Some("/home/ammar/junk/rust-dwarf-x86/test_files/simple_executable")).expect("Failed to load r2");
+    r2.analyze();
+    let src: Rc<Source> = Rc::new(Rc::new(RefCell::new(r2)));
+    let mut p = ProjectLoader::default()
+        .path("/home/ammar/junk/rust-dwarf-x86/test_files/simple_executable")
+        .source(Rc::clone(&src))
+        .module_loader(ModuleLoader::default()
+                        .parallel()
+                        .build_ssa()
+                        .build_callgraph()
+                        .load_datarefs()
+                        .function_loader(FunctionLoader::default().include_defaults()))
+        .load();
+
+    let regfile = SubRegisterFile::default();
```

Comment:
Don't do default cause it won't have any information inside it. You need to get this from `RadecoProject`